packages/util/src/util.ts

```diff
@@ -0,0 +1,12 @@
+export type PromiseOrValue<T> = Promise<T> | T;
+
+export const resolveObjectOfPromises = async <T>(obj: {
```

Comment:
How about this name?
```suggestion
export const resolveObjectValues = async <T>(obj: {
```

Reply:
60b094607 should solve this